source/jormungandr/tests/disruptions_tests.py

```diff
@@ -85,6 +86,9 @@ def test_disruption_end_point(self):
         eq_(network_disrupt[0]['disruption_id'], 'disruption_on_line_A')
         eq_(network_disrupt[0]['uri'], 'too_bad_again')
 
+
+        eq_(hasattr(network_disrupt[1], 'contributor'), False)
```

Comment:
hasattr works for `dict` too ?

I would have done `'contributor' not it network_disrupt[1]`